src/language-js/embed/utils.js

```diff
@@ -80,6 +80,9 @@ function hasLanguageComment({ node, parent }, languageName) {
   return (
     hasLeadingBlockCommentWithName(node, languageName) ||
     (isAsConstExpression(parent) &&
+      hasLeadingBlockCommentWithName(parent, languageName)) ||
+    (parent.type === "ExpressionStatement" &&
+      node.type === "TemplateLiteral" &&
       hasLeadingBlockCommentWithName(parent, languageName))
```

Comment:
```suggestion
    (parent.type === "ExpressionStatement" &&
      hasLeadingBlockCommentWithName(parent, languageName))
```

Reply:
Thanks - I think I realized that check wasn't strictly necessary, but I just don't know enough about how things work here, so in my mind I was playing it safe somehow. I'll update this today.

Reply:
Also, would a better description be: `Format embedded GQL in template literal statements`?